### PR TITLE
Replace indirect download links

### PR DESCRIPTION
**Pull request by lava**
The client-side redirects on the `vast.io` domain would send command-line clients into an endless redirect loop. Therefore, replace these links by their resolved targets.

Fixes tenzir/issues#288.

### CODE REVIEW

File: web/docs/setup/download.md [hunk 1]
```diff
@@ -14,15 +14,15 @@ release and the current development version.
 ### Generic Linux
 
 <div align="center" class="padding-bottom--md">
-  <a class="button button--md button--primary margin-right--md" href="/download/vast-linux-static.tar.gz">Static Build (Release)</a>
-  <a class="button button--md button--info margin-left--md" href="/download/vast-linux-static-latest.tar.gz">Static Build (Development)</a>
+  <a class="button button--md button--primary margin-right--md" href="https://github.com/tenzir/vast/releases/latest/download/vast-linux-static.tar.gz">Static Build (Release)</a>
```
Comment (dominiklohmann) [inline web/docs/setup/download.md:17]: Do we want to keep the redirects in `web/src/pages/download/*.tsx` around, or delete them as well?

**Review by dominiklohmann — APPROVED** [review]

File: web/docs/setup/download.md (hunk 1 above)
Comment (mavam) [reply web/docs/setup/download.md:17]: I would say remove them.

Comment (lava) [reply web/docs/setup/download.md:17]: That's going to break the version in `versioned_docs`, do we want to retro-actively replace the links there as well?

Comment (dominiklohmann) [reply web/docs/setup/download.md:17]: For this case I would say yes.

**Pull request merged**